qgreenland/tasks/common/shapefile.py

```diff
@@ -67,7 +67,24 @@ def run(self):
             input_filename = shapefile
 
         with temporary_path_dir(self.output()) as temp_path:
-            infile = os.path.join(self.input().path, input_filename)
+            if 'makevalid' in ogr2ogr_kwargs:
```

Comment:
Should we just do this `makevalid` step for all vector layers that use `ogr2ogr`?

Reply:
:+1: